recipes/minikube/meta.yaml

```
@@ -0,0 +1,51 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "minikube" %}
+{% set version = "v1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kubernetes/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 7aa57e5896852c499f1687fbc424abf93645e1801fc9f8c2833e0affbb76eb41
+  patches:
+    - patch/go.sum.patch
+    - patch/go.mod.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - go
```

I think this should use the compiler syntax:
```suggestion
    - {{ compiler("go") }}
```

---

oh I understood now what @sodre said about this ... thanks for the explanation!